app/src/pages/outside/loginPage/serviceVersionsBlock/serviceVersionsBlockWithData.jsx

```diff
@@ -36,22 +36,38 @@ export class ServiceVersionsBlockWithData extends Component {
   calculateServices = (serviceVersions, latestServiceVersions) => {
     const services = {};
 
-    Object.keys(serviceVersions).map((objKey) => {
-      const value = serviceVersions[objKey];
-      const currentVersion = value.build.version;
-      const latestVersion = latestServiceVersions[value.build.repo];
-
-      services[objKey] = {
-        name: value.build.name,
-        version: value.build.version,
-        newVersion: latestVersion || null,
-        repo: value.build.repo || null,
-        isDeprecated:
-          !!value.build.repo && !!latestVersion && !!semverDiff(currentVersion, latestVersion),
-      };
+    Object.keys(serviceVersions).map((serviceKey) => {
```

Comment:
Format of returning data will be changed on backend side. So for now this PR should be on hold.

Reply:
It won't be changed. Some versions are already exixt on this endpoint, so this task can be done without waiting for WS